cpp/log/cert.cc

```diff
@@ -1128,30 +1130,34 @@ Cert::Status TbsCertificate::DeleteExtension(int extension_nid) {
 
   // ExtensionIndex returns the first matching index - if the extension
   // occurs more than once, just give up.
-  int ignored;
-  status = ExtensionIndex(extension_nid, &ignored);
-  if (status == Cert::TRUE) {
+  const StatusOr<int> ignored_index = ExtensionIndex(extension_nid);
```

Comment:
Using assignment instead of constructor syntax, but no worries, it's all the same...

I just dislike the idea of "assigning to a `const` object", and how this triggers only the copy constructor (or even constructing in place, if RVO kicks in), short-circuiting any `operator=`.